### ISSUE TITLE
Aria-value text for molecules AB and ABC

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/molecule-polarity#217]
- [x] The aria-value text for rotating the molecule should be in degrees not hours.  

 accessibleObjectResponse for Molecule AB  and Molecule ABC (ARIA valuetext for slider): 
0 degrees
30 degrees
60 degrees
90 degrees
120 degrees
150 degrees
180 degrees
210 degrees
240 degrees
270 degrees
300 degrees
330 degrees
360 degrees

_Note: _ have to use ) instead of 360 because the same angle can't be given 2 different values. 

- [x] Also,  it would be nice to hear the same context response when Molecule ABC rotates as we hear on screen 1:

accessibleObjectResponse: 
On initial arrow press for interaction with slider and on direction change: {{Rotation2}}

- [x] You had asked about this response a while back: "Rotating {{Rotation2}}. Horizontal, aligned with electric field." Is it easy to separate it into 2 context responses?
      Rotating {{Rotation2}}.
_Then once the molecule stops:_  Aligned with electric field.

**Comment by Nancy-Salpepi** [comment]
- [ ] When rotating the molecule using shift + arrow, please have the aria value text update. Right now as I rotate,  0 degrees gets repeated until I hit 30 degrees. 

- [ ] When the molecule slowly, responses are repeated:

https://github.com/user-attachments/assets/981d4e76-be18-4f06-8d63-a871924235bc
